src/NuGet.Config

```diff
@@ -7,7 +7,7 @@
     <add key="myget.org dotnet-coreclr" value="https://www.myget.org/F/dotnet-coreclr/" />
     <add key="myget.org dotnet-corefx" value="https://www.myget.org/F/dotnet-corefx/" />
     <add key="myget.org dotnet-buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
-    <add key="myget.org aspnetvnext" value="https://www.myget.org/F/aspnetvnext/" />
+    <add key="myget.org nugetbuild" value="https://www.myget.org/F/nugetbuild/" />
```

Comment:
This is a new feed that the nuget team is publish their packages to (the nuget.*.dlls).  Previously they were on aspnetvnext, but that was incorrect.